### DISCUSSION TITLE
Correct setup with Nextcloud?

### DISCUSSION THREAD

**Discussion by BurningTheSky** [photoprism/photoprism#1881] [Q&A]
Hi all,

I have Nextcloud as my source of truth for my photos as I use the Nextcloud app to upload new photos from my phone, what is the correct way to mount the Nextcloud photos folder in Photoprism so that it will automatically show new images that have been uploaded?

I'm using Unraid and currently have the files mounted to the storage path but that doesn't seem to work that way, it needs a manual reindex to work. Do I need to mounth the folder somewhere else or is there a setting I'm missing to get auto working?

Thanks!

**Comment by graciousgrey** [comment]
You can mount the nextcloud photos folder to PhotoPrism's originals directory (see https://docs.photoprism.app/getting-started/docker-compose/#volumes).

The index process is triggered automatically after photos have been added using webdav (with a safety delay) --> https://docs.photoprism.app/user-guide/library/originals/#automatic-indexing. 
In case the nextcloud app is not using webdav to sync files from your phone, you can configure the scheduler to automatically index files --> https://dl.photoprism.app/docker/scheduler/

**Comment by BurningTheSky** [comment]
I will have a look to see if Nextcloud is using webdav, if it is do I have to "link" nextcloud and photoprism so that nextcloud informs photoprism?

**Reply by graciousgrey** [reply]
If your nextcloud photos folder is mounted as PhotoPrism's originals folder and nextcloud uses webdav it should work without further configuration.

**Reply by BurningTheSky** [reply]
In that case I'm guessing either Nextcloud isn't using webdav or Unraid isn't mounting it as the originals folder. I can see the file structure but maybe I need to use a different variable in Unraid. Alternatively, do you know how to use the job.ini file with Unraid? I don't have a compose yaml to edit that I'm aware of

**Reply by graciousgrey** [reply]
Does indexing files in PhotoPrism work (https://docs.photoprism.app/user-guide/library/originals/) ? 
Maybe this video can help you: https://www.youtube.com/watch?v=WMNsO-0BuG8&t=8s

**Reply by BurningTheSky** [reply]
Yeah, I can manually trigger the index without issue. I've seen that video before any it doesn't help unfortunately. It's just a basic setup which I've already done, doesn't discuss scheduling the index process.
